konoha/__init__.py

```diff
@@ -1,5 +1,7 @@
 """__init__.py."""
 from konoha.sentence_tokenizer import SentenceTokenizer  # NOQA
 from konoha.word_tokenizer import WordTokenizer  # NOQA
+from importlib_metadata import version
 
-__version__ = "4.6.3"
+
+__version__ = version("konoha")
```

Comment:
Use `importlib_metadata` to get a current version.
After flair drops support for Python3.6, I'll replace `importlib_metadata` with a built-in `importlib.metadata`.